lib/chai/core/assertions.js

```diff
@@ -1197,7 +1215,7 @@ module.exports = function (chai, _) {
   /**
    * ### .below(n[, msg])
    *
-   * Asserts that the target is a number less than the given number `n`.
+   * Asserts that the target is a number or a date less than the given number or a date `n` respectively.
```

Comment:
@v1adko In almost every case, the suggestion is "given number or date" instead of "given number or a date". In addition to that, the one for `.above` is also missing an "a" at the start.